thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-wizard.component.tsx

```diff
@@ -12,214 +12,162 @@
-                    }
-                );
-            }
-            setSelectedAlertsGroup(updated);
-        };
+        useEffect(() => {
+            const newSubscriptionGroupAssociations: AlertAssociation[] =
+                editedAssociations.map(({ alertId, enumerationId }) => ({
+                    alert: { id: alertId },
+                    ...(enumerationId && {
+                        enumerationItem: { id: enumerationId },
+                    }),
+                }));
+
+            // Remove the @deprecated `alerts` key from existing data
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            setEditedSubscriptionGroup(({ alerts, ...stateProp }) => ({
+                ...stateProp,
+                alertAssociations: newSubscriptionGroupAssociations,
+            }));
+        }, [editedAssociations]);
+
+        const isSubscriptionGroupValid = validateSubscriptionGroup(
+            editedSubscriptionGroup
+        );
 
         return (
             <>
-                <PageContentsGridV1>
-                    <Grid item xs={12}>
-                        <PageContentsCardV1>
-                            <Typography variant="h5">
-                                {t("label.subscription-group-properties")}
-                            </Typography>
-                            <Typography variant="body2">
-                                {t("message.basic-information-about-group")}
-                            </Typography>
-                            <Box marginTop={3}>
-                                <SubscriptionGroupPropertiesForm
-                                    subscriptionGroup={editedSubscriptionGroup}
-                                    onChange={handleSubscriptionChange}
-                                />
-                            </Box>
-                        </PageContentsCardV1>
-                    </Grid>
-
-                    <Grid item xs={12}>
-                        <PageContentsCardV1>
-                            <Typography variant="h5">
-                                {t("label.channels")}
-                            </Typography>
-                            <Typography variant="body2">
-                                {t("message.channels-subtitle")}
-                            </Typography>
-                            <Box marginTop={3}>
-                                <GroupsEditor
-                                    subscriptionGroup={editedSubscriptionGroup}
-                                    onSpecsChange={handleSpecsChange}
-                                    onSubscriptionGroupEmailsChange={
-                                        onSubscriptionGroupEmailsChange
-                                    }
-                                />
-                            </Box>
-                        </PageContentsCardV1>
-                    </Grid>
+                <PageHeader {...pageHeaderProps} />
```

Comment:
What header to display should be handled by respective parent pages and should be rendered there as well.

Reply:
Moved the header out.

My initial thought for keeping the header inside was to reduce the amount of duplication required to set up the props for the page header (and the tabs in it), but it does make sense to let the parent component decide how and what tab should be shown, for future extensibility.